### PR TITLE
Issue 163 Render probability fix

### PR DESCRIPTION
**Pull request by Kasmadei**
@blcham

### CODE REVIEW

**Comment by Kasmadei** [comment]
Not sure why here is update for README

**Comment by blcham** [comment]
It is ok like that, we can talk about it on standup ... 

It is because akaene:main was not merged into kbss-cvut:main yet ... if you would want to see only one commit you would have to do merge akaene:main to kbss-cvut:main first.

File: README.md [hunk 1]
```diff
@@ -10,7 +10,8 @@ It is primarily focused on FTA which offers possibilities for system partonomy d
 ```shell script
 npm install
 npm run build
-npm run preview
+npm run preview (only preview)
+npm run dev (for development)
```
Comment (LaChope) [inline README.md:14]: We do not need these comments

**Review by LaChope — APPROVED** [review]
Just need to remove the comments from read.me, otherwise LGTM

File: README.md (hunk 1 above)
Comment (Kasmadei) [reply README.md:14]: When i was trying to start project first for time, i did not find correct command for it

Comment (LaChope) [reply README.md:14]: Sorry, I meant the commands are correct, just not what's between the parenthesis

Comment (LaChope) [reply README.md:14]: Also, if we add `npm run dev`, we should specify that we should use either `npm run install && npm run dev` **or** `npm run install && npm run build && npm run preview`

Comment (Kasmadei) [reply README.md:14]: How do u start the project ? I was using npm install && npm run dev. Is it correct ?

Comment (Kasmadei) [reply README.md:14]: Also here is one failing check, not sure how can i fix it

Comment (blcham) [reply README.md:14]: I hopefully fixed failing checks so when you commit next it should be fixed.

**Comment by blcham** [comment]
@Kasmadei I tried to merge, but there is a conflict now which I believe you can resolve easily. If you merge akaene:main to this branch, it should be resolved.

File: README.md (hunk 1 above)
Comment (blcham) [reply README.md:14]: @Kasmadei I addressed your need to have some development notes in https://github.com/kbss-cvut/fta-fmea-ui/pull/191

Comment (blcham) [reply README.md]: resolved in #191

**Pull request merged**